mappings/net/minecraft/test/GameTest.mapping

```diff
@@ -3,10 +3,20 @@ CLASS net/minecraft/unmapped/C_qanvkdqp net/minecraft/test/GameTest
 	COMMENT
 	COMMENT <p>{@code GameTest} methods must take 1 parameter of {@link net.minecraft.test.TestContext}
 	METHOD m_balpyuta batchId ()Ljava/lang/String;
+		COMMENT The id of the batch this test should belong to
 	METHOD m_hdcufkgx required ()Z
+		COMMENT Whether this test must succeed for the whole test sequence to succeed
 	METHOD m_hxdvfvrg structureName ()Ljava/lang/String;
+		COMMENT An {@link net.minecraft.util.Identifier} describing the location of the structure file to load for this test
+		COMMENT
+		COMMENT <p>The actual path for the file depends on the current test framework, but usually gets resolved as {@code "<namespace>:gametest/structures/<location>.nbt"}
 	METHOD m_mpjcinpl maxAttempts ()I
-	METHOD m_nntnqpde duration ()J
+		COMMENT How many times the test can be restarted
```

Comment:
I think this can be a bit more explicit, mentioning that the test is being restarted because it fails. Maybe something like `How many times the test can fail and be restarted` (not that, because the wording sounds silly, but you get the gist)

Reply:
Fair enough, tried to improve the explanation a bit